spec/controllers/quotes_controller_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe QuotesController, type: :controller do
+  let(:quote) { create(:quote) }
+
+  describe "#new" do
+    subject { get :new }
+    it "renders the new template" do
+      expect(subject).to render_template(:new)
+    end
+
+    it "instantiates a new Quote variable" do
+      subject
+      expect(assigns(:quote)).to be_a_new Quote
+    end
+  end
+
+  describe "#create" do
+    context "with valid parameters" do
+      subject { post :create, quote: attributes_for(:quote) }
+      it "changes the quotes count by +1" do
+        expect { subject }.to change { Quote.count }.by(1)
+      end
+
+      it "redirects to the show page" do
+        expect(subject).to redirect_to quote_path(Quote.last)
+      end
+    end
+    context "with invalid parameters" do
+      subject { post :create, quote: attributes_for(:quote).merge(body: nil) }
+      it "renders the new template filled" do
+        expect(subject).to render_template(:new)
+      end
+
+      it "doesn't change the quotes count" do
+        expect { subject }.to_not change { Quote.count }
+      end
+    end
+  end
+
+  describe "#edit" do
+  end
+
+  describe "#show" do
+    subject { get :show, id: quote.id }
+    it "renders the show template" do
+      expect(subject).to render_template(:show)
+    end
+
+    it "instantiates an instance variable with the passed id" do
+      subject
+      expect(assigns(:quote)).to eq(quote)
+    end
+  end
+
+  describe "#index" do
+    subject { get :index }
+    it "renders the index template" do
+      expect(subject).to render_template(:index)
+    end
+
+    context "more than one quote in database" do
+      let!(:quote_1) { create(:quote) }
+      before { quote }
+      it "instantiates a quotes variable containing all quotes" do
+        subject
+        expect(assigns(:quotes)).to eq([quote_1, quote])
```

Comment:
I don't think creating an additional quote is necessary here, the behaviour isn't any different then with a single record.
can just be 

`expect(assigns(:quotes)).to eq([quote])`

then you can get rid of the context.

In general the usage of `context` is implying that there is some kind of branching behaviour, if I don't see a context, and then another that is complementary I'm going to be very confused.